comp/core/autodiscovery/listeners/snmp_deduper.go

```diff
@@ -0,0 +1,188 @@
+// Unless explicitly stated otherwise all files in this repository are licensed
+// under the Apache License Version 2.0.
+// This product includes software developed at Datadog (https://www.datadoghq.com/).
+// Copyright 2025-present Datadog, Inc.
+
+package listeners
+
+import (
+	"math"
+	"net"
+	"sync"
+
+	"github.com/DataDog/datadog-agent/pkg/snmp"
+	"github.com/DataDog/datadog-agent/pkg/util/log"
+)
+
+const uptimeDiffTolerance = 50
```

Comment:
- Add the unit, like 50 * time.Milliseconds
- 50ms seems a tiny bit small to be. What about 5 seconds ?